deps/CMakeLists.txt

```diff
@@ -14,17 +10,9 @@ function(mapbox_base_add_library name include_path)
         ${include_path}
     )
 
-    target_link_libraries(mapbox-base INTERFACE mapbox-base-${name})
+    # target_link_libraries(mapbox-base INTERFACE mapbox-base-${name})
 endfunction()
 
-add_library(mapbox-base INTERFACE)
-add_library(Mapbox::Base ALIAS mapbox-base)
-
-execute_process(
```

Comment:
Please don't remove this during this transitional period - we need it so we make git submodule management as part of the build system flow.

Reply:
Yes, but in future we need to avoid git usage from cmake